compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/bir/codegen/JvmMethodGen.java

```diff
@@ -1493,6 +1500,14 @@ void generateLambdaMethod(BIRInstruction ins, ClassWriter cw, String lambdaName)
         mv.visitEnd();
     }
 
+    private List<BType> getInitialParamTypes(List<BType> paramTypes, int argsCount) {
```

Comment:
try to remove functionwrapper. 
Use only symbol.

Reply:
we can't completely remove the function wrapper. Because, when building the pack, the langlib modules are not updated in the symbol maps until they are compiled. So only relying on symbols can cause errors